chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/Id.java

```diff
@@ -0,0 +1,29 @@
+package com.griddynamics.jagger.user.test.configurations;
```

Comment:
Propose to move all auxiliary classes to separate package
/configurations/aux or /configurations/helpers